src/auth.ts

```diff
@@ -132,24 +132,21 @@ export class Role {
  * be added to Roles to manage permissions.
  */
 export class User {
-  constructor(
-    private client: AuthClient,
-    public readonly name: string,
-  ) {}
+  constructor(private client: AuthClient, public readonly name: string) {}
 
   /**
    * Creates the user, with the provided password.
    */
   public create(password: string): Promise<this> {
-    return this.client.userAdd({ name: this.name, password })
-      .then(() => this);
+    return this.client.userAdd({ name: this.name, password }).then(() => this);
```

Comment:
So, I actually meant to comment on this for a while now, but why resolve `this`? This seems to be a pretty niche use case as you usually have the variable already in the scope when using this.